pkg/live/example-resource-group-crd-for-v1.yaml

```diff
@@ -80,8 +80,31 @@ spec:
                     type: string
                 type: object
               resources:
-                description: Resources contains a list of resources that form the
-                  resource group
+                description: Resources contains a list of resources that form the resource
+                  group
+                items:
+                  description: ObjMetadata organizes and stores the identifying information
+                    for an object. This struct (as a string) is stored in a grouping
+                    object to keep track of sets of applied objects.
+                  properties:
+                    group:
+                      type: string
```

Comment:
Do we need `GroupKind` for this? Is there any situation where the subgroup wouldn't be a ResourceGroup?

Reply:
The subgroup should always be ResourceGroup. The reason of adding this is for simplicity in the controller so that it handles the subgroups and resources in the same way.

Reply:
I'll remove the unnecessary `GroupKind` in the controller code. Will update this CRD afterwards.

Reply:
Updated this. PTAL